zingo-testutils/src/lib.rs

```diff
@@ -576,49 +585,12 @@ pub mod scenarios {
         )
     }
 
-    pub async fn two_wallet_one_synced_orchard_transaction(
-        value: u64,
-        regtest_network: zingoconfig::RegtestNetwork,
-    ) -> (
-        RegtestManager,
-        ChildProcessHandler,
-        LightClient,
-        LightClient,
-        String,
-    ) {
-        dbg!("0 About to create faucet_recipient.");
-        let (regtest_manager, child_process_handler, faucet, recipient) =
-            two_wallet_one_miner_fund(regtest_network).await;
-        dbg!("1 About to increase height and sync faucet.");
-        increase_height_and_wait_for_client(&regtest_manager, &faucet, 1)
-            .await
-            .unwrap();
-        dbg!("2 faucet synced.");
-        let txid = faucet
-            .do_send(vec![(
-                &get_base_address!(recipient, "unified"),
-                value,
-                None,
-            )])
-            .await
-            .unwrap();
-        dbg!("3 faucet send complete");
-        increase_height_and_wait_for_client(&regtest_manager, &recipient, 1)
-            .await
-            .unwrap();
-        dbg!("4 recipient increased and synced.");
-        dbg!("5 about to sync faucet.");
-        faucet.do_sync(false).await.unwrap();
-        (
-            regtest_manager,
-            child_process_handler,
-            faucet,
-            recipient,
-            txid,
-        )
+    pub async fn faucet_default() -> (RegtestManager, ChildProcessHandler, LightClient) {
+        let regtest_network = zingoconfig::RegtestNetwork::all_upgrades_active();
+        faucet(regtest_network).await
     }
 
-    pub async fn two_wallet_one_miner_fund(
+    pub async fn faucet_recipient(
```

Comment:
Can we make this something like `faucet_recipient_network`?

Reply:
as you will see in the next PR (mining to orchard) these scenarios will not be limited by parameters. regtestnetwork is just the first of many parameters we think would be useful for setting up scenarios. i think default calling a general version makes more sense than using the general name by default and adding for example `_custom` to the general version. also custom_clients would become custom_clients_custom hehe